Assets/Scripts/Ui/Achivment.cs

```diff
@@ -0,0 +1,76 @@
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+using System;
+
+namespace TankBattle.UI
+{
+    public class Achivment : MonoBehaviour
+    {
+        private Animator popUp;
+        private TextMeshProUGUI[] achievementText;
+        private Rank rank;
+        private int enemyCount = 0;
+        private int shootCount = 0;
+
+        private void Start()
+        {
+            popUp = GetComponent<Animator>();
+            achievementText = GetComponentsInChildren<TextMeshProUGUI>();
```

Comment:
Do not access the child component like this, drag and drop it. There can be multiple child objects containing same objects which can create problems in getting the correct reference.